### PR TITLE
[Runtime] Document registering your own runtime template

### PR DESCRIPTION
**Pull request by theofidry**

### CODE REVIEW

**Comment by alamirault** [comment]
What do you think to had a link to https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Runtime/Internal/autoload_runtime.template in order to show an example and/or the default runtime template ?

**Comment by theofidry** [comment]
👍 mind doing a PR to it?

File: components/runtime.rst [hunk 1]
```diff
@@ -504,5 +504,28 @@ The end user will now be able to create front controller like::
         return new SomeCustomPsr15Application();
     };
 
+Create Your Own Runtime Template
+--------------------------------
+
+In some cases the default generic runtime template generated (`vendor/autoload_runtime.php`)
+file is not enough. For example: you want to use a different entry point than
+`$_SERVER['FILE_SCRIPTNAME']` or a different SAPI than the traditional `cli`,
+`phpdbg` or `embed`.
```
Comment (OskarStark) [inline components/runtime.rst]:
```suggestion
``$_SERVER['FILE_SCRIPTNAME']`` or a different SAPI than the traditional ``cli``,
``phpdbg`` or ``embed``.
```

File: components/runtime.rst [hunk 2]
```diff
@@ -504,5 +504,28 @@ The end user will now be able to create front controller like::
         return new SomeCustomPsr15Application();
     };
 
+Create Your Own Runtime Template
```
Comment (theofidry) [inline components/runtime.rst]: after discussing, we want to try to put it a tip in "usage".

**Comment by javiereguiluz** [comment]
This was finally merged. Thanks Théo.

**Pull request merged**